PowerEditor/src/NppDarkMode.cpp

```diff
@@ -364,7 +373,13 @@ namespace NppDarkMode
 		initExperimentalDarkMode();
 		initAdvancedOptions();
 
-		if (NppDarkMode::isWindowsModeEnabled() && NppDarkMode::isWindows10())
+		g_isAtLeastWindows10 = NppDarkMode::isWindows10();
```

Comment:
adjusted code from https://github.com/notepad-plus-plus/notepad-plus-plus/pull/12466
optimize checks for windows mode